docs/architecture/containerized-lifecycle/design-develop-containerized-apps/deploy-azure-kubernetes-service.md

```diff
@@ -1,64 +1,64 @@
 ---
 title: Orchestrating microservices and multi-container applications for high scalability and availability
 description: Learn how to deploy an app using Azure Kubernetes Service.
-ms.date: 02/15/2019
+ms.date: 08/06/2020
 ---
-# Deploy to Azure Kubernetes Service (AKS)
-
-You can interact with AKS using your preferred client operating system, here we show how to do it with Microsoft Windows and an embedded version of Ubuntu Linux in Windows, using Bash commands.
 
-Prerequisites to have before using AKS are:
+# Deploy to Azure Kubernetes Service (AKS)
 
-- Linux or Mac development machine
-- Windows development machine
-  - Developer Mode enabled on Windows
-  - Windows Subsystem for Linux
-- Azure-CLI installed on [Windows, Mac or Linux](https://docs.microsoft.com/cli/azure/install-azure-cli)
+You can interact with AKS using your preferred client operating system (Microsoft Windows, Mac and Linux). You also need to have either a Web Browser or Azure command-line interface(Azure CLI) installed in your development machine. For more details, see [Install the Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)
 
 > [!NOTE]
 > To find complete information about:
 >
-> Azure-CLI: <https://docs.microsoft.com/cli/azure/index>
->
-> Windows Subsystem for Linux: <https://docs.microsoft.com/windows/wsl/about>
+> Azure CLI: <https://docs.microsoft.com/cli/azure/index>
```

Comment:
Remove this line since we have this covered in the earlier para.

Reply:
Removed.